Tests/LibC/TestLibCInodeWatcher.cpp

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/API/InodeWatcherEvent.h>
+#include <Kernel/API/InodeWatcherFlags.h>
+#include <LibTest/TestCase.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <utime.h>
+
+constexpr unsigned maximum_event_size = sizeof(InodeWatcherEvent) + NAME_MAX + 1;
```

Comment:
Shouldn't this be a constant exported by the `UnixTypes.h` header or LibC, it seems very strange to make usermode compute this themselves.

Reply:
Added to `InodeWatcherEvent.h` since that's where this is used anyway.